tests/foreman/cli/test_activationkey.py

```diff
@@ -204,9 +202,13 @@ def test_positive_create_content_and_check_enabled(module_org, module_target_sat
         {'url': settings.repos.yum_0.url, 'organization-id': module_org.id}
     )
     content = module_target_sat.cli.ActivationKey.product_content(
-        {'id': result['activationkey-id'], 'organization-id': module_org.id}
+        {
+            'id': result['activationkey-id'],
+            'organization-id': module_org.id,
+            'content-access-mode-all': 1,
+        }
     )
-    assert content[0]['default-enabled?'] == 'true'
+    assert content[0]['default-enabled?'] == 'false'
```

Comment:
Custom content is disabled by default in 6.15, see https://docs.redhat.com/en/documentation/red_hat_satellite/6.15/html-single/managing_content/index#Restricting_Hosts_Access_to_Content_content-management